File: python/x.py
l1 = None
def hello(arr):
    length = len(arr)
    subarray: list = None
    for i in range(length):
        for j in range(i,length):
            subarray.append(arr[i:j+1])
    return subarray

l1 = [2,3,4,5]

result = hello(l1)
for m in result:
    print(m)

File: more c codes/x.c
#include <stdio.h>

int list[100][100],subarray[100],length,i,j,m;

void sub(int arr[],int length){
    int count=0;


    for (i=0;i < length; i++){
        for (j=0; j < i; j++){
            int outercount = 0;
            for (m=i;m <= j; m++){
                list[count][outercount++]=arr[m];
            }
            subarray[count] = outercount;
            count++;
        }
        

    }
}

